internal/db/sample_models.go

```diff
@@ -74,15 +81,36 @@ var Profile = Model{
 }
 
 var Post = Model{
-	Id:   uuid.MustParse("e25750c8-bb31-41fe-bdec-6bff1dceb2b4"),
+	ID:   uuid.MustParse("e25750c8-bb31-41fe-bdec-6bff1dceb2b4"),
 	Name: "post",
 	Attributes: map[string]Attribute{
 		"text": Attribute{
-			Id:       uuid.MustParse("b3af6694-b621-43a2-be7f-00956fa505c0"),
-			AttrType: String,
+			ID:       uuid.MustParse("b3af6694-b621-43a2-be7f-00956fa505c0"),
+			Datatype: String,
 		},
 	},
 	RightRelationships: []Relationship{
 		UserPosts,
 	},
 }
+
+var Andrew = Datatype{
+	ID:            uuid.MustParse("46c0ee11-3943-452d-9420-925dd9be8208"),
+	Name:          "andrew",
+	Validator:     AndrewValidator,
+	StorageFormat: StringFormat,
+}
+
+var AndrewValidator = Code{
+	ID:       uuid.MustParse("aaea187b-d153-4c4a-a7e7-cda599b02ba6"),
+	Name:     "andrewValidator",
+	Runtime:  Starlark,
+	Function: Validator,
+	Code: `
+def func():
```

Comment:
I wonder if we could just invoke func with arg and get its return value (so the code in func can be more natural). e.g. append a statement at the end during execution like `RETURN_VALUE = func(arg)`

Reply:
Let's chat through this one at the end of the review. I don't love how it works now and certainly can be improved